sentry-launchdarkly-android/src/main/java/io/sentry/launchdarkly/android/SentryLaunchDarklyAndroidHook.java

```diff
@@ -0,0 +1,70 @@
+package io.sentry.launchdarkly.android;
+
+import com.launchdarkly.sdk.EvaluationDetail;
+import com.launchdarkly.sdk.LDValue;
+import com.launchdarkly.sdk.LDValueType;
+import com.launchdarkly.sdk.android.integrations.EvaluationSeriesContext;
+import com.launchdarkly.sdk.android.integrations.Hook;
+import io.sentry.IScopes;
+import io.sentry.ScopesAdapter;
+import io.sentry.SentryIntegrationPackageStorage;
+import io.sentry.SentryLevel;
+import io.sentry.util.IntegrationUtils;
+import java.util.Map;
+import java.util.Objects;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class SentryLaunchDarklyAndroidHook extends Hook {
+  private final IScopes scopes;
+
+  static {
+    SentryIntegrationPackageStorage.getInstance()
+        .addPackage("maven:io.sentry:sentry-launchdarkly-android", BuildConfig.VERSION_NAME);
+  }
+
+  public SentryLaunchDarklyAndroidHook() {
+    this(ScopesAdapter.getInstance());
+  }
+
+  public SentryLaunchDarklyAndroidHook(final @NotNull IScopes scopes) {
+    super("SentryLaunchDarklyAndroidHook");
+    this.scopes = Objects.requireNonNull(scopes, "Scopes are required");
+    addPackageAndIntegrationInfo();
+  }
+
+  private void addPackageAndIntegrationInfo() {
+    IntegrationUtils.addIntegrationToSdkVersion("LaunchDarkly-Android");
+  }
+
+  @Override
+  public Map<String, Object> afterEvaluation(
```

Comment:
Since this is a public method anyway, we could write a test which mimics the `afterEvaluation` call, and verifies a mocked scope `.addFeatureFlag()` call is being made. WDYT?